README.md

```diff
@@ -240,21 +240,23 @@ services:
     container_name: immich_server
     volumes:
      - /path/to/my/photos:/external_libs/photos
-     - /path/to/secret/file:/immich_api_key.secret:ro
   ...
   immich-folder-album-creator:
     container_name: immich_folder_album_creator
     image: salvoxia/immich-folder-album-creator:latest
     restart: unless-stopped
+    volumes:
+     - /path/to/secret/file:/immich_api_key.secret:ro
     environment:
       API_URL: http://immich_server:2283/api
       API_KEY_FILE: /immich_api_key.secret
       ROOT_PATH: /external_libs/photos
       CRON_EXPRESSION: "0 * * * *"
       TZ: Europe/Berlin
-
 ```
 
+This will periodically re-scan the library as per `CRON_EXPRESSION` settings. To trigger re-scanning manually (even with a different environment), you can run e.g. `docker exec immich_folder_album_creator /bin/sh -c "UNATTENDED=1 MODE=CLEANUP /script/immich_auto_album.sh"` on command line for the running container.
```

Comment:
The `docker exec ...` example contains `MODE=CLEANUP`, which might be misleading in the context of the README file.
Would you adjust the example to omit the `MODE` variable?

Reply:
Yeah, it was a bad example.

On the second thought:

If the user has setup Docker Compose for the album creator with a cron job, he already has the UNATTENDED=1 in the environment, doesn't he?

Otherwise, he would have album creator running every few hours with no effect on the album creation at all.

Would you know a better example of setting a variable for `docker exec ...`?

Reply:
Crontab is set up to explicitly pass `UNATTENDED=1` before calling the script, just like in the  `docker exec ...` example (refer to https://github.com/Salvoxia/immich-folder-album-creator/blob/f6c6e7e5164f72ab1c06cc1187d876a8d89164df/docker/setup_cron.sh#L4).
I suggest to add two examples to the README file: One for performing the dry-run and one for actually creating the albums on demand in an already running container.